torch/nn/modules/module.py

```diff
@@ -462,6 +463,9 @@ def __init__(self) -> None:
         super().__setattr__('_load_state_dict_post_hooks', OrderedDict())
         super().__setattr__('_modules', OrderedDict())
 
+        if Module.call_super_init:
```

Comment:
You should use `self.call_super_init` here such that subclasses can override the behavior just for them without changing the global Module behavior!

Reply:
ok,
https://github.com/pytorch/pytorch/pull/91819/commits/03b24eb3c90bf51af38eade867f2d1d8c531200e